README.md

```diff
@@ -1,3 +1,5 @@
+**Note: This repository is being largely rewritten from the ground up on the `reorganize` branch. Minor issues on the `master` bring will not be given high priority, and new contributors may wish to base their work on that branch.**
```

Comment:
IMO you should do this like this:

- Open a PR from `reorganize` to `master`. Call it "WIP - reorganize" (the WIP is a symbol that it's not merge-worthy yet regardless what your CI says)
- People coming to the repo continue to PR into `master`
- You keep pushing on `reorganize` and regularly rebase to current `master`

I think that would be a lot cleaner contributor experience (assuming that "being largely rewritten" means you pushing on master, not many collaborators). Even if you don't want to do that, this note should give an estimated timeline for when `master` will be `master` again.

Reply:
> You keep pushing on reorganize and regularly rebase to current master

That's a generally good idea, but there's no easy way to make my changes incrementally, and the hard way isn't worthwhile given that we don't have a lot of contributors and I have to make the most of the limited windows of time I have to work on this project.

> Even if you don't want to do that, this note should give an estimated timeline for when `master` will be `master` again.

I can't make any promises beyond that I'll try to make the most of the limited time I have. We released this project before it was really ready because I wanted to be able to keep working on it. After the overhaul it will be a lot more usable, I'll start doing more to promote it, and at that point I'll worry more about the contribute experience.